aspnetcore/security/authorization/secure-data.md

```diff
@@ -67,6 +67,13 @@ This tutorial is advanced. You should be familiar with:
 
 [Download](xref:index#how-to-download-a-sample) the [completed](https://github.com/dotnet/AspNetCore.Docs/tree/main/aspnetcore/security/authorization/secure-data/samples) app. [Test](#test-the-completed-app) the completed app so you become familiar with its security features.
 
+> [!TIP]
+> To only clone the subfolder, use [`git sparse-checkout`](https://git-scm.com/docs/git-sparse-checkout) command as follows:
+> git clone --depth 1 --filter=blob:none https://github.com/dotnet/AspNetCore.Docs.git --sparse
+> cd AspNetCore.Docs
+> git sparse-checkout init --cone
+> git sparse-checkout set aspnetcore/security/authorization/secure-data/samples
```

Comment:
>

Reply:
Hi  @Rick-Anderson was there a comment to resolve please?